src/ChangeSet.ts

```diff
@@ -62,9 +62,19 @@ export class ChangeSet {
    */
   get changeTree() {
     const rootNodes: TrackedChange[] = []
+    const referenceChanges = new Map<string, number>()
```

Comment:
please provide a comment here so that'd be clearer in future what it's for

Reply:
updated